doc/source/docs/reference/sindex.rst

```diff
@@ -39,11 +39,11 @@ methods:
 The concrete implementations currently available are
 ``geopandas.sindex.PyGEOSSTRTreeIndex`` and ``geopandas.sindex.RTreeIndex``.
 
-In addition to the methods listed above, the ``rtree``-based spatial index
+In addition to the methods listed above, the rtree-based spatial index
```

Comment:
I would keep this. We need to distinguish between RTree as a structure of an index and `rtree` as a library providing an implementation. It should be clear that this sentence refers to the latter.